sinkSender.go

```diff
@@ -40,59 +43,51 @@ const failureText = `Unfortunately, your endpoint is not able to keep up with th
 // FailureMessage is a helper that lets us easily create a json struct to send
 // when we have to cut and endpoint off.
 type FailureMessage struct {
-	Text string `json:"text"`
-	// Original     ancla.InternalWebhook `json:"webhook_registration"` //TODO: add back in once ancla/argus dependency issue is fixed.
-	CutOffPeriod string `json:"cut_off_period"`
-	QueueSize    int    `json:"queue_size"`
-	Workers      int    `json:"worker_count"`
+	Text         string       `json:"text"`
+	Original     ListenerStub `json:"webhook_registration"` //TODO: remove listener stub once ancla/argus issues fixed
+	CutOffPeriod string       `json:"cut_off_period"`
+	QueueSize    int          `json:"queue_size"`
+	Workers      int          `json:"worker_count"`
```

Comment:
we should consider if keeping the original struct makes sense... like if `Original     ListenerStub` is not used then we should get rid of it

just a general suggestion